content/en/news/kubesphere-2021-year-in-review.md

```diff
@@ -0,0 +1,119 @@
+
+![KubeSphere Community Report 2021](/images/news/community-report-2021/project-metrics.png)
+
+## User Adoption
+
+According to telemetry statistics, KubeSphere has been downloaded 70w times worldwide, with nearly 8w active clusters and thousands of enterprises using KubeSphere in production. We've seen KubeSphere adopted by users in a variety of industries, including banking, insurance, securities, internet, energy, telecom, transportation, universities, retail, logistics, etc., some experienced users who have helped the KubeSphere project by submitting suggestions or feedback and contributing to the code and documentation. 
+
+In addition, we see an increasing number of users evangelizing and promoting KubeSphere at community events, technical articles, and on social media.
+
+![KubeSphere Community Report 2021](/images/news/community-report-2021/known-adopters.png)
+
+## Known Contributing Enterprises and Partners
+
+We are grateful to see more and more enterprises contributing to KubeSphere and its ecosystem projects in the past year. At the same time, we have built partnership with the fast-growing enterprises and technical giants to create joint solutions and enrich the application ecosystem. For example, six cloud-native applications (APISIX, ChaosMesh, Nocalhost, JuiceFS, EdgeMesh, fpga-operator) had been added to  KubeSphere App Store by our partners.
+
+
+![KubeSphere Community Report 2021](/images/news/community-report-2021/contributing-enterprises-partners.png)
+
+## Integration with Cloud Providers
+
+As a uified control plane for multi-cloud and multi-cluster management, KubeSphere can easily import and manage private and hosted Kubernetes clusters. In 2021, KubeSphere is available on AWS, Azure, DigitalOcean, and QingCloud, integrating Amazon EKS, Azure AKS, DigitalOcean Kubernetes, and QingCloud QKE, respectively.
+
+- [KubeSphere for Amazon EKS](https://aws.amazon.com/quickstart/architecture/qingcloud-kubesphere/)
+- [KubeSphere for Azure AKS](https://market.azure.cn/marketplace/apps/qingcloud.kubesphere)
+- [KubeSphere for DigitalOcean Kubernetes](https://marketplace.digitalocean.com/apps/kubesphere)
+- [KubeSphere on QingCloud AppCenter(QKE)](https://www.qingcloud.com/products/kubesphereqke)
+
+![KubeSphere Community Report 2021](/images/news/community-report-2021/integration-with-cloud.png)
+
+## KubeSphere Member
+
+KubeSphere Members are granted to members who have deeply participated in the open-source contribution to the KubeSphere community. These members have independently developed or tested one or more features, written or translated documents, or helped to promote KubeSphere and open-source technologies in various communities. Now we have invited all of them to join KubeSphere GitHub.
```

Comment:
`open-source technologies`:
I see you use `open source` in other places, such as `open source projects` in line 17. How about using either `open source` or `open-source` throughout this article?